pom.xml

```diff
@@ -165,6 +173,16 @@
             <timeoutSeconds>30</timeoutSeconds>
             <timeoutRetries>2</timeoutRetries>
           </systemProperties>
+          <includes>
+            <include>**/Test*.java</include>
+            <include>**/*Test.java</include>
+            <include>**/*Tests.java</include>
+            <include>**/*TestCase.java</include>
+            <include>AnnotatedBaseTestClass.java</include>
```

Comment:
I don't think we want to run this, its an abstract class so it can't be instantiated anyway (I'm surprised Surefire doesn't complain). As long as we're running the classes which extend it we're fine.